app/frontend/wallet/helpers/tokenFormater.ts

```diff
@@ -39,3 +38,8 @@ export const groupTokensByPolicyId = (tokens: Token[]): {[policyId: string]: Tok
     .groupBy(({policyId}) => policyId)
     .value()
 }
+
+export const getTokenDifference = (tokens1: Token[], tokens2: Token[]): Token[] => {
```

Comment:
related to the suggestion above, I'd name this `getTokenBundlesDifference`

Reply:
Fixed.